tests/test_contracts/test_class_contracts.py

```diff
@@ -147,6 +168,38 @@ def test_change_age_invalid_in_method(person) -> None:
     assert age == 10
 
 
+def test_change_age_of_other_invalid_in_method(person, person_2) -> None:
+    """
+    Call a method that changes age of another instance of the same class to something invalid but
+    back to something valid.
+    Expects normal behavior.
+    """
+    age = person.decrease_and_increase_others_age(person_2, 10)
+    assert age == 10
+
+
+def test_violate_ri_in_other_instance(person, person_2) -> None:
+    """
+    Call a method that changes age of another instance of the same class to something invalid
+    Excepts the RI to be violated hence an AssertionError to be thrown.
```

Comment:
"Excepts" -> "Expects"

"thrown" -> "raised"

Also, please add a period to the end of the sentence on the previous line.

Reply:
Done!